gnovm/cmd/gno/lint_test.go

```diff
@@ -11,15 +11,19 @@ func TestLintApp(t *testing.T) {
 			args:                []string{"lint", "--set_exit_status=0", "../../tests/integ/run-main/"},
 			stderrShouldContain: "./../../tests/integ/run-main: missing 'gno.mod' file (code=1).",
 		}, {
-			args:                []string{"lint", "--set_exit_status=0", "../../tests/integ/run-main/"},
-			stderrShouldContain: "./../../tests/integ/run-main: missing 'gno.mod' file (code=1).",
+			args:                []string{"lint", "--set_exit_status=0", "../../tests/integ/undefined-variable-test/undefined_variables_test.gno"},
+			stderrShouldContain: "undefined_variables_test.gno:6: name toto not declared (code=2)",
+		}, {
+			args:                []string{"lint", "--set_exit_status=0", "../../tests/integ/package-not-declared/main.gno"},
+			stderrShouldContain: "main.gno:4: name fmt not declared (code=2).",
```

Comment:
I used to replace those hard to read tests by testscript like in #1103.
 If we can prevent the coverage lost by using specific flags like @moul mentionned :
 
 ```
 https://go.dev/testing/coverage/

Building a binary for coverage profiling
$ go build -cover -o myprogramMorePkgs.exe -coverpkg=io,mydomain.com,rsc.io/quote .
```

Reply:
> If we can prevent the coverage lost by using specific flags like @moul mentioned

totally, if this prevent coverage lost, it will be awesome !